nano/node/node.cpp

```diff
@@ -975,8 +975,12 @@ void nano::node::ongoing_unchecked_cleanup ()
 
 void nano::node::ongoing_backlog_population ()
 {
-	populate_backlog ();
+	auto overflow = populate_backlog ();
 	auto delay = config.network_params.network.is_dev_network () ? std::chrono::seconds{ 1 } : std::chrono::duration_cast<std::chrono::seconds> (std::chrono::minutes{ 5 });
+	if (overflow)
```

Comment:
Is setting delay to 0 when there is overflown bucket really indented? The improvements @RickiNano wrote about are because it happens that it's always true, so we never wait thus constantly overfilling some buckets and in effect never starving aec. Not starving aec is good but definitely can be done more efficiently by using shorter delay time and doing backlog population when accounts queued in scheduler drop below some threshold.

<img width="903" alt="aecbeforeafter" src="https://user-images.githubusercontent.com/3044353/149606206-4196e82a-b44e-40d4-b0c2-d7f7b0f13db0.png">